firrtl/src/main/scala/firrtl/ir/Serializer.scala

```diff
@@ -17,7 +17,7 @@ object Serializer {
   val Indent = "  "
 
   // The version supported by the serializer.
-  val version = Version(4, 2, 0)
+  val version = Version(5, 1, 0)
```

Comment:
Most code paths are probably going through the Chisel IR serializer. Hopefully we can delete this one soonish (part of the motivation for https://github.com/chipsalliance/chisel/pull/4878)